psa-crypto/src/types/key.rs

```diff
@@ -844,16 +874,15 @@ impl From<UsageFlags> for psa_crypto_sys::psa_key_usage_t {
         if flags.decrypt {
             usage_flags |= psa_crypto_sys::PSA_KEY_USAGE_DECRYPT;
         }
-        //TODO: not yet implemented in Mbed Crypto, uncomment when added
-        //if flags.sign_message {
-        //usage_flags |= psa_crypto_sys::PSA_KEY_USAGE_SIGN_MESSAGE;
-        //}
+        if flags.sign_message {
+            usage_flags |= psa_crypto_sys::PSA_KEY_USAGE_SIGN_MESSAGE;
+        }
         if flags.sign_hash {
             usage_flags |= psa_crypto_sys::PSA_KEY_USAGE_SIGN_HASH;
         }
-        //if flags.verify_message {
-        //usage_flags |= psa_crypto_sys::PSA_KEY_USAGE_VERIFY_MESSAGE;
```

Comment:
That was commented because the Mbed TLS version used by this crate does not support it, that's why the tests fail I think 😢
Very topical as this PR (https://github.com/ARMmbed/mbedtls/pull/4357) seems to add it! Meanwhile I think this is fine to keep this commented.
The whole permissions around sign/very hash/message will need to be revamped as part of #82 anyway